docs/docs/pyproject.md

```diff
@@ -243,15 +243,32 @@ mysqlclient = { version = "^1.3", optional = true }
 [tool.poetry.extras]
 mysql = ["mysqlclient"]
 pgsql = ["psycopg2"]
+databases = ["mysqlclient", "psycopg2"]
 ```
 
-When installing packages, you can specify extras by using the `-E|--extras` option:
+
```

Comment:
Maybe move this section after the `poetry` specific documentation.

Reply:
Just so I am clear, do you mean move this to [section](https://python-poetry.org/docs/pyproject/#poetry-and-pep-517)?

Reply:
I should have been clearer, sorry. I was proposing that your additions go blow what already exists as opposed to above it. So your new lines regarding `x[a]` usage go after the existing documentation around using `poetry install -E`.